network-api/networkapi/wagtailpages/pagemodels/research_hub/authors_index.py

```diff
@@ -84,29 +84,29 @@ def get_author_detail_context(self, profile_id: int):
             id=profile_id,
         )
 
-        author_research_count = self.get_author_research(author_profile).count()
-        latest_research = self.get_latest_research(author_profile)
-
-        # On author detail pages to include the link to the authors index.
-        detail_page_breadcrumbs = self.get_breadcrumbs(include_self=True)
-
         return {
             "author_profile": author_profile,
-            "author_research_count": author_research_count,
-            "breadcrumbs": detail_page_breadcrumbs,
-            "latest_research": latest_research,
+            "author_research_count": self.get_author_research_count(author_profile=author_profile),
+            # On author detail pages to include the link to the authors index.
+            "breadcrumbs": self.get_breadcrumbs(include_self=True),
+            "latest_research": self.get_latest_author_research(author_profile=author_profile),
             "library_page": library_page.ResearchLibraryPage.objects.first(),
         }
```

Comment:
The intermediate variables just seemed quite useless since we were not doing anything with them.